app/src/test/java/com/example/parkingproyect/ParkingReservationTest.java

```diff
@@ -0,0 +1,226 @@
+package com.example.parkingproyect;
+
+import com.example.parkingproyect.entities.Parking;
+import com.example.parkingproyect.entities.Reservation;
+import com.example.parkingproyect.mvp.model.ReservationModel;
+import com.example.parkingproyect.mvp.presenter.ReservationPresenter;
+import com.example.parkingproyect.mvp.view.ReservationView;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+
+import java.util.Calendar;
+import java.util.Date;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(RobolectricTestRunner.class)
+public class ParkingReservationTest {
+    private ReservationPresenter presenter;
+    private ReservationModel model;
+    private ReservationView view;
+
+    @Before
+    public void setup() {
+        model = mock(ReservationModel.class);
+        view = mock(ReservationView.class);
+        presenter = new ReservationPresenter(model, view);
+    }
+
+    @Test
+    public void validate_show_start_date_time() {
```

Comment:
There are several naming conventions for tests, but our proposal is to use: MethodName_StateUnderTest_ExpectedBehavior.
For example, this should be renamed to onStartDateTimePressed_ showEndDateTimeInput.
Apply the same convention for all tests.